stories/features/newTab/components/page/style.ts

```diff
@@ -0,0 +1,35 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styled from 'styled-components'
+import theme from '../../../../../src/theme/brave-default'
+import palette from '../../../../../src/theme/palette'
+
+export const StyledPage = styled<{}, 'div'>('div')`
+  -webkit-font-smoothing: antialiased;
+  background: linear-gradient(
+    ${theme.color.privateTabBackground},
```

Comment:
See comment about in brave-default theme module. Since this is not a core component, and the theme name is not generalized, and since we don't have any of the other colors for this page in the theme, and _also_ since this would never change between light / dark mode, we can just inline the colors.

Another reason to not use theme: it's not using the color palette either.